src/passes/CoalesceLocals.cpp

```diff
@@ -569,7 +569,7 @@ void CoalesceLocalsWithLearning::pickIndices(std::vector<Index>& indices) {
 #ifdef CFG_LEARN_DEBUG
   std::cout << "[learning for " << getFunction()->name << "]\n";
 #endif
-  auto numVars = getFunction()->getNumVars();
+  auto numVars = this->getFunction()->getNumVars();
```

Comment:
hmm, what is special about this location (and the few others)? what does VS need `this->` here but not in 99% of the other places `this` is implicitly used?